post/urls.py

```diff
@@ -1,11 +1,13 @@
 from django.urls import path
 from . import views as post_views
-
+from post.views import AddCommentView 
 app_name = "post"
 
 urlpatterns = [
     path("<int:id>/", post_views.post, name="post"),
     path("add/", post_views.add_post, name="add-post"),
     path("<int:id>/delete/", post_views.delete_post, name="delete-post"),
     path("<int:id>/update/", post_views.update_post, name="update-post"),
+    path("<int:pk>/comment/",AddCommentView.as_view(), name="add_comment"),
```

Comment:
Can we stick here with convention that we use '-' instead of '_' for name? :)

Reply:
right, didn't notice that one

Reply:
if

Reply:
What kind of error? Have you changed the name in templates in {% url '<namespace>:<name>' %} as well?